result/capture.yaml

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
what is the expectation of capture result? what is the format of the returned packets?  i wonder if it is sufficient to get back a link to pcap, so that the client can use other tools to dissect/interpret as they wish?

Reply:
That's a good point - but should it be a two step process with specific transport implications (ie http link) or just return the pcap as application/octet-stream (binary). Also the start/stop is on multiple ports, the get results is on multiple ports - so the structure being returned can be array of port name and pcap binary (array of objects allows for model growth with no bw compat issues)